From d029f7751a29da3ba8eec2c05b3087753d5fde5c Mon Sep 17 00:00:00 2001
Message-ID: <d029f7751a29da3ba8eec2c05b3087753d5fde5c.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 01:09:41 +0100
Subject: [PATCH 15/19] gnu: go-github-com-onsi-ginkgo: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-github-com-onsi-ginkgo):
Move from here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 32 ++++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 28 ----------------------------
 2 files changed, 32 insertions(+), 28 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index eb4f085bbf..ce4afe64ac 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -7,6 +7,7 @@
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2021 Tobias Geerinckx-Rice <me@tobias.gr>
+;;; Copyright © 2022 ( <paren@disroot.org>
 ;;; Copyright © 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
 ;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
@@ -274,6 +275,37 @@ (define-public go-github-com-jacobsa-ogletest
 messages automatically.")
       (license license:asl2.0))))
 
+(define-public go-github-com-onsi-ginkgo
+  (package
+    (name "go-github-com-onsi-ginkgo")
+    (version "1.16.5")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/onsi/ginkgo")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32
+         "1hh6n7q92y0ai8k6rj2yzw6wwxikhyiyk4j92zgvf1zad0gmqqmz"))))
+    (build-system go-build-system)
+    (arguments
+     (list #:import-path "github.com/onsi/ginkgo"))
+    (propagated-inputs
+     (list go-golang-org-x-sys
+           go-golang-org-x-tools
+           go-github-com-go-task-slim-sprig
+           go-github-com-nxadm-tail
+           go-github-com-onsi-gomega))
+    (home-page "https://github.com/onsi/ginkgo")
+    (synopsis "BDD-style testing framework for Go")
+    (description
+     "Ginkgo is a Behaviour-Driven Development testing framework for Go.  It
+builds on top of Go's builtin @code{testing} library and is complemented by the
+Gomega matcher library.")
+    (license license:expat)))
+
 (define-public go-github-com-stretchr-testify
   (package
     (name "go-github-com-stretchr-testify")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 5abe17bf3e..3ff544a67b 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -13615,34 +13615,6 @@ (define-public go-github-com-onsi-gomega
      "Gomega is the preferred matcher library for the Ginkgo test framework.")
     (license license:expat)))
 
-(define-public go-github-com-onsi-ginkgo
-  (package
-    (name "go-github-com-onsi-ginkgo")
-    (version "1.16.5")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/onsi/ginkgo")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1hh6n7q92y0ai8k6rj2yzw6wwxikhyiyk4j92zgvf1zad0gmqqmz"))))
-    (build-system go-build-system)
-    (arguments (list #:import-path "github.com/onsi/ginkgo"))
-    (propagated-inputs (list go-golang-org-x-sys
-                             go-golang-org-x-tools
-                             go-github-com-go-task-slim-sprig
-                             go-github-com-nxadm-tail
-                             go-github-com-onsi-gomega))
-    (home-page "https://github.com/onsi/ginkgo")
-    (synopsis "BDD-style testing framework for Go")
-    (description
-     "Ginkgo is a Behaviour-Driven Development testing framework for Go.  It
-builds on top of Go's builtin @code{testing} library and is complemented by the
-Gomega matcher library.")
-    (license license:expat)))
-
 ;;;
 ;;; Avoid adding new packages to the end of this file. To reduce the chances
 ;;; of a merge conflict, place them above by existing packages with similar
-- 
2.41.0

